run_unit_tests.sh

```diff
@@ -61,8 +61,13 @@ if [ -f "${AIRFLOW_DB}" ]; then
 fi
 
 # For impersonation tests on Travis, make airflow accessible to other users via the global PATH
-# (which contains /usr/local/bin)
-sudo ln -sf "${VIRTUAL_ENV}/bin/airflow" /usr/local/bin/
+# (which contains /usr/local/bin).  Some test environments, like the docker instructions, won't have sudo and
+# are probably running as root anyway
+if which sudo > /dev/null; then
```

Comment:
'which' is not the best way as . Better/portable way is `command -v`

Reply:
Nice tip, thanks!